app/views/payments/_form.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= form_with(model: [@project, @payment], remote: true, html: { role: "form" } ) do |form| %>
```

Comment:
do we need remote: true with form_with?

Reply:
Yes Sir as we are submitting form through ajax, we need it